M/MLX/bundled/patches/cmake_system_processor-arm64-aarch64.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 949c891e..15f9a6a0 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -57,7 +57,7 @@ else()
+   message(WARNING "MLX is prioritised for Apple silicon systems using macOS.")
+ endif()
+ 
+-if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm64")
++if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
```

Comment:
This patch should probably be upstreamed (maybe with an OR for both arm64 and aarch64)

Reply:
Right

Reply:
https://github.com/ml-explore/mlx/pull/1626

Reply:
I don't think this has anything to do with cross-compilation, it's just that different systems report different names.